source-zendesk-support/source_zendesk_support/source.py

```diff
@@ -127,20 +138,25 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             TicketFields(**args),
             TicketMetrics(**args),
             TicketMetricEvents(**args),
+            TicketSkips(**args),
             Tickets(**args),
             Users(**args),
             Brands(**args),
             CustomRoles(**args),
             Schedules(**args),
         ]
+        # AccountAttributes, AttributeDefinitions, and TicketForms streams are only available for Enterprise Plan users 
+        # but the Zendesk API does not provide a public API to get user's subscription plan. That's why we try to read 
+        # at least one record and expose this stream in case of success or skip it otherwise
         ticket_forms_stream = TicketForms(**args)
-        # TicketForms stream is only available for Enterprise Plan users but Zendesk API does not provide
-        # a public API to get user's subscription plan. That's why we try to read at least one record and expose this stream
-        # in case of success or skip it otherwise
         try:
             for stream_slice in ticket_forms_stream.stream_slices(sync_mode=SyncMode.full_refresh):
                 for _ in ticket_forms_stream.read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream_slice):
-                    streams.append(ticket_forms_stream)
```

Comment:
Is it possible to have an Enterprise plan and have 0 records come back from `TicketForms`? In that case `TicketForms` wouldn't be added, and neither would `AccountAttributes` or `AttributeDefinitions`.

Reply:
It is not possible to have an Enterprise plan & have no `TicketForms` records. Every Enterprise plan starts with a default ticket form, and the default ticket form can't be deleted. I tested & confirmed this behavior with API calls.